### DISCUSSION TITLE
Welcome to fvtt-syrin-control Discussions!

### DISCUSSION THREAD

**Discussion by frondeus** [frondeus/fvtt-syrin-control#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChrisEurolog** [comment]
How to add the ambient sounds from Syrinscape? I see from the screenshot on the module page you were able to enter the wav file name - I cannot remember how to do this in Syrinscape (LOL!) and cannot find how to do this online through Google etc. - all help appreciated :)
